src/main/resources/META-INF/plugin.xml

```diff
@@ -1,39 +1,53 @@
 <idea-plugin url="snyk.io">
     <id>io.snyk.snyk-intellij-plugin</id>
     <name>Snyk Vulnerability Scanning</name>
-    <vendor url="http://snyk.io">Snyk.io</vendor>
+    <vendor url="http://snyk.io">Snyk Ltd.</vendor>
     <version>${project.version}</version>
 
     <description><![CDATA[
-    <html>
-        <p>
-            This is an IDE extension that helps you detect and fix security issues in your project.
-        </p>
-        <p>
-            By scanning against the vast Snyk database, we can identify and advise
-            you how to fix security problems in your open source dependencies.
-        </p>
-    </html>
+    <p>
+        The Snyk Vulnerability Scanner plugin for IntelliJ helps you find and fix
+        security vulnerabilities in your projects, all from within your favorite IDE.
+    </p>
+    <br/>
+    <p>
+        The plugin integrates seamlessly into your development environment and scans the open source
+        dependencies included in your IntelliJ projects for security vulnerabilities. Identified
+        vulnerabilities are displayed with actionable information, including the full dependency path
+        and remediation advice to help you fix issues as fast as possible.
+    </p>
+
+    <h2><strong>Key features:</strong></h2>
+    <ul>
+        <li>Seamless integration</li>
+        <li>Accurate results based on Snyk Intel</li>
```

Comment:
Is `Snyk Intel` a thing? Asking because of the capitalization of Intel.

Reply:
it's a text from marketing team, so yes. should be so.